src/client/components/Navigation/Topnav.less

```diff
@@ -158,6 +158,25 @@
   }
 }
 
+div[data-dir='rtl'] .Topnav {
+  &__menu-container {
+    display: flex;
```

Comment:
I don't think you need to declare, `display: flex` here since it's already declared at the regular `.Topnav` styles

Reply:
Thanks, redundant indeed.